test/Microsoft.AspNetCore.Mvc.Razor.Extensions.Test/TestFiles/IntegrationTests/CodeGenerationIntegrationTest/Inherits.cshtml

```diff
@@ -0,0 +1,2 @@
+﻿@using RazorCodeGen
+@inherits MyBasePage<string> base
```

Comment:
Do you just mean that they should both be TModel and one of them should have `@page`? If not I'm not sure I follow since the models we're inheriting from don't exist.